GitVersion.yml

```diff
@@ -1,5 +1,4 @@
 mode: ContinuousDeployment
-next-version: 14.0
```

Comment:
@michaelnoonan Adding the +semver:major works
https://github.com/OctopusDeploy/Calamari/pull/586/commits/b31ade9d0610e35140bdb965c9c926fae34d164e

Reply:
That looks good. I can't comment on the rest. :)